go.mod

```diff
@@ -59,6 +59,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1
 )
 
+require gotest.tools v2.2.0+incompatible
```

Comment:
Any reason to add this? Isn't gomega assert good to use?

Reply:
I think I commented by mistake to use Gomega which is used in E2E tests. I was really meaning to use [testify library](https://pkg.go.dev/github.com/stretchr/testify@v1.8.4/assert) which is used in unit testing instead. Apologies for the confusion.